src/Menu/Menu.js

```diff
@@ -2,6 +2,7 @@ import Callbacks from "../classes/Callbacks";
 import UI from "../General/UI";
 import { g, Conf } from "../globals/globals";
 import $ from "../platform/$";
+import Iocn from "../Icons/icon";
```

Comment:
You spelled Icon wrong.

![image](https://github.com/TuxedoTako/4chan-xt/assets/126348645/af5ab39a-9b12-49fc-8f8e-b8576c503599)

What editor are you using? VS code warns about this:

![Code_8JZvQV24ns](https://github.com/TuxedoTako/4chan-xt/assets/126348645/39f08d39-e7e1-4264-b56a-ce64ee92466f) (spell checker is an extension)
![Code_oVnv70eCR4](https://github.com/TuxedoTako/4chan-xt/assets/126348645/9a6eb12f-cb87-488f-a544-c5a7dd7ca051)